cherrypy/test/test_static.py

```diff
@@ -97,6 +97,21 @@ def bytesio(self):
                 f = io.BytesIO(b'Fee\nfie\nfo\nfum')
                 return static.serve_fileobj(f, content_type='text/plain')
 
+            @cherrypy.expose
+            def serve_file_utf8_filename(self):
+                file_path = os.path.join(curdir, 'style.css')
+                return static.serve_file(
+                    file_path,
```

Comment:
How about just serving this test module?
```suggestion
                return static.serve_file(
                    __file__,
```